tsconfig.json

```diff
@@ -9,14 +9,14 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "resolveJsonModule": true,
-    "target": "es5",
+    "target": "es2020",
     "sourceMap": true,
     "noEmit": false,
     "incremental": true,
     "strict": true,
     "baseUrl": "./", // this must be specified if "paths" is specified.
     "rootDir": "./",
-    "lib": ["ES2015", "ES2022.Error"],
```

Comment:
we can remove es2015 from https://github.com/PostHog/posthog-js-lite/blob/a1d9091396682de005f448d11531c0425e0b18d5/posthog-web/tsconfig.json#L4 as well

Reply:
Also here https://github.com/PostHog/posthog-js-lite/blob/a1d9091396682de005f448d11531c0425e0b18d5/jest.config.js#L18